css/contacts.toolbar.css

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * Styling for the contacts module toolbar.
+ */
+
+/**
+ * Toolbar tab icon.
+ */
+.toolbar-bar .toolbar-icon-page-manager-page-view-contacts-dashboard:before {
+  background-image: url(../images/contacts.png);
+}
+.toolbar-bar .toolbar-icon-page-manager-page-view-contacts-dashboard:active:before,
+.toolbar-bar .toolbar-icon-page-manager-page-view-contacts-dashboard.active:before {
+  background-image: url(../images/contacts.png);
+}
+
+.toolbar-bar .toolbar-icon-edit:before {
+  background-color: #bebebe;
+  mask: url(../images/pencil.svg) no-repeat 50% 50%;
+  -webkit-mask: url(../images/pencil.svg) no-repeat 50% 50%;
```

Comment:
Masks?!

Reply:
Yup. `fill` doesnt work if the SVG is a `background-image`. But this does the trick.